### PR TITLE
add firewall manager resources

### PR DESCRIPTION
**Pull request by tylersouthwick**

### CODE REVIEW

**Review by der-eismann — DISMISSED** [review]

**Comment by svenwltr** [comment]
Hello,

could you please add some properties? See https://github.com/rebuy-de/aws-nuke/blob/master/CONTRIBUTING.md#use-properties-instead-of-string-functions

**Review by bjoernhaeuser — DISMISSED** [review]

File: resources/fms_policies.go [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+	return err
+}
+
+func (f *FMSPolicy) String() string {
+	return *f.policy.PolicyId
+}
+
+func (f *FMSPolicy) Properties() types.Properties {
+	properties := types.NewProperties()
+	properties.Set("PolicyId", f.policy.PolicyId)
```
Comment (svenwltr) [inline resources/fms_policies.go:66]:
```suggestion
	properties.Set("PolicyID", f.policy.PolicyID)
```

This does not follow the Golang naming conventions. Abbreviations should have consistent case. See [Guidelines](https://github.com/golang/go/wiki/CodeReviewComments#initialisms).

**Review by svenwltr — COMMENTED** [review]
Sorry, this is the last nitpick :)

File: resources/fms_policies.go (hunk 1 above)
Comment (tylersouthwick) [reply resources/fms_policies.go:66]: we could set the property as `PolicyID` but, the field name from the aws sdk is `PolicyId`

Comment (bjoernhaeuser) [reply resources/fms_policies.go]: I've updated this quickly. We try to stick with golang names as far as we can. We may have missed some though.

**Review by bjoernhaeuser — APPROVED** [review]

**Pull request merged**